accounts/abi/bind/template.go

```diff
@@ -0,0 +1,205 @@
+}
+
+{{range .Calls}}
+	{{if .Structured}}
+		// {{.Normalized.Name}}Result is the result of the {{.Normalized.Name}} invocation."
+		type {{.Normalized.Name}}Result struct {
+				{{range .Normalized.Outputs}}{{.Name}} {{bindtype .Type}}
+				{{end}}
+		}
+	{{end}}
+
+	// {{.Normalized.Name}} is a free data retrieval call binding the contract method 0x{{printf "%x" .Original.Id}}.
+	//
+	// Solidity: {{.Original.String}}
+	func (_{{$.Type}} *{{$.Type}}Caller) {{.Normalized.Name}}(opts *bind.CallOpts {{range .Normalized.Inputs}}, {{.Name}} {{bindtype .Type}} {{end}}) ({{if .Structured}}*{{.Normalized.Name}}Result,{{else}}{{range .Normalized.Outputs}}{{bindtype .Type}},{{end}}{{end}} error) {
+		var (
+			{{if .Structured}}ret = new(*{{.Normalized.Name}}Result){{else}}{{range $i, $_ := .Normalized.Outputs}}ret{{$i}} = new({{bindtype .Type}})
+			{{end}}{{end}}
+		)
+		out := {{if .Structured}}ret{{else}}{{if eq (len .Normalized.Outputs) 1}}ret0{{else}}[]interface{}{
+			{{range $i, $_ := .Normalized.Outputs}}ret{{$i}},
+			{{end}}
+		}{{end}}{{end}}
+		err := _{{$.Type}}.contract.Call(opts, out, "{{.Original.Name}}" {{range .Normalized.Inputs}}, {{.Name}}{{end}})
+		return {{if .Structured}}*ret,{{else}}{{range $i, $_ := .Normalized.Outputs}}*ret{{$i}},{{end}}{{end}} err
+	}
+
+	// {{.Normalized.Name}} is a free data retrieval call binding the contract method 0x{{printf "%x" .Original.Id}}.
+	//
+	// Solidity: {{.Original.String}}
+	func (_{{$.Type}} *{{$.Type}}Session) {{.Normalized.Name}}({{range $i, $_ := .Normalized.Inputs}}{{if ne $i 0}},{{end}} {{.Name}} {{bindtype .Type}} {{end}}) ({{if .Structured}}*{{.Normalized.Name}}Result, {{else}} {{range .Normalized.Outputs}}{{bindtype .Type}},{{end}} {{end}} error) {
+	  return _{{$.Type}}.Contract.{{.Normalized.Name}}(&_{{$.Type}}.CallOpts {{range .Normalized.Inputs}}, {{.Name}}{{end}})
+	}
+
+	// {{.Normalized.Name}} is a free data retrieval call binding the contract method 0x{{printf "%x" .Original.Id}}.
+	//
+	// Solidity: {{.Original.String}}
+	func (_{{$.Type}} *{{$.Type}}CallerSession) {{.Normalized.Name}}({{range $i, $_ := .Normalized.Inputs}}{{if ne $i 0}},{{end}} {{.Name}} {{bindtype .Type}} {{end}}) ({{if .Structured}}*{{.Normalized.Name}}Result, {{else}} {{range .Normalized.Outputs}}{{bindtype .Type}},{{end}} {{end}} error) {
+	  return _{{$.Type}}.Contract.{{.Normalized.Name}}(&_{{$.Type}}.CallOpts {{range .Normalized.Inputs}}, {{.Name}}{{end}})
+	}
+{{end}}
+
+{{range .Transacts}}
+	// {{.Normalized.Name}} is a paid mutator transaction binding the contract method 0x{{printf "%x" .Original.Id}}.
+	//
+	// Solidity: {{.Original.String}}
+	func (_{{$.Type}} *{{$.Type}}Transactor) {{.Normalized.Name}}(opts *bind.TransactOpts {{range .Normalized.Inputs}}, {{.Name}} {{bindtype .Type}} {{end}}) (*types.Transaction, error) {
+		return _{{$.Type}}.contract.Transact(opts, "{{.Normalized.Name}}" {{range .Normalized.Inputs}}, {{.Name}}{{end}})
```

Comment:
Should `{{.Normalized.Name}}` not be `{{.Original.Name}}` when looking up the correct function?